patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback.twig

```diff
@@ -0,0 +1,100 @@
+<div class="ma__mass-feedback-form">
+  <h2 class="visually-hidden">Feedback</h2>
+
+  <form class="ma__mass-feedback-form__form" method="post" action="{{ formAction}}" id="">
```

Comment:
Maybe we should have options to pass in the id? Maybe even make novalidate optional based on a flag?